### PR TITLE
Issue selected

### PR DESCRIPTION
**Pull request by ebrehault**

### CODE REVIEW

File: projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.html [hunk 1]
```diff
@@ -1,33 +1,43 @@
...
-        </ng-container>
-        <div class="button-container">
-            <pa-button class="select-all"
-                       size="small"
-                       color="secondary"
-                       (click)="toggleSelectAll()">
-                {{selectAllLabel}}
-            </pa-button>
+        </div>
+        <div class="counts-and-actions">
```
Comment (mathilde-pellerin) [inline projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.html]: `pa-` prefix

File: projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.html [hunk 2]
```diff
@@ -1,33 +1,43 @@
...
+            <div>
+                <div class="total">
```
Comment (mathilde-pellerin) [inline projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.html]: `pa` prefix

File: projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.ts [hunk 3]
```diff
@@ -150,10 +150,25 @@ export class FilteredCheckboxGroupComponent implements OnInit, OnChanges, OnDest
     }
 
     private applyFilter() {
-        this.filtered = this._checkboxes.filter(ctl => (!this.filter && !this.selectedLetter) || ctl.isFiltered);
+        this.filtered = this._checkboxes.filter(ctl => ((!this.filter && !this.selectedLetter) || ctl.isFiltered) &&
+            (this.viewAll || ctl.isSelected));
```
Comment (mathilde-pellerin) [inline projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.ts]: this part is really hard to understand, even though I'm pretty sure it's right. could you add a comment to clarify it?

Comment (ebrehault) [reply projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.ts]: fixed

**Pull request merged**